src/test/java/com/avparamonov/checkers/services/GameServiceTest.java

```diff
@@ -27,47 +25,73 @@
+    @Before
+    public void setUp() {
+        player1 = playerService.create("Tom", PlayerType.HUMAN, Side.WHITE);
+        player2 = playerService.create("Jerry", PlayerType.COMPUTER, Side.BLACK);
+    }
 
     @Test
     public void testCreateGame() throws Exception {
-        Player player1 = Player.builder()
-                .nickname("Player1")
-                .age(25)
-                .type(PlayerType.HUMAN)
-                .side(Side.BLACK)
-                .build();
-        playerRepository.save(player1);
-
-        Player player2 = Player.builder()
-                .nickname("Player2")
-                .type(PlayerType.COMPUTER)
-                .side(Side.RED)
-                .build();
-        playerRepository.save(player2);
-
-        gameService.createGame(player1, player2);
-
-        List<Game> games = gameService.findAll();
-        List<Player> players = (List<Player>) playerRepository.findAll();
-        List<Board> boards = (List<Board>) boardRepository.findAll();
-        List<Cell> cells = (List<Cell>) cellRepository.findAll();
-        List<Checker> checkers = (List<Checker>) checkerRepository.findAll();
-
-        Assert.assertEquals(1, games.size());
-        Assert.assertEquals(2, players.size());
-        Assert.assertEquals(1, boards.size());
-        Assert.assertEquals(64, cells.size());
-        Assert.assertEquals(24, checkers.size());
+        game = gameService.createGame("Tom", "Jerry");
+
+    }
+
+    @Test
+    public void testMakeRegularJump() throws Exception {
+        game = gameService.createGame("Tom", "Jerry");
+
+        Checker[][] board = game.getBoard();
+
+        for (int row = 0; row < 8; row++) {
+            for (int col = 0; col < 8; col++) {
+                board[row][col] = null;
+            }
+        }
+        board[2][4] = Checker.builder().row(2).col(4).side(Side.WHITE).type(CheckerType.REGULAR).build();
+        board[3][3] = Checker.builder().row(3).col(3).side(Side.BLACK).type(CheckerType.REGULAR).build();
+
+        Assert.assertNotNull(board[3][3]);
+
+        Move jump = new Move(2, 4, 4, 2);
+        gameService.makeMove(player1, jump, game);
+
+        Assert.assertNull(board[3][3]);
+    }
+
+    @Test
+    public void testMakeKingJump() throws Exception {
+        game = gameService.createGame("Tom", "Jerry");
+
+        Checker[][] board = game.getBoard();
+
+        for (int row = 0; row < 8; row++) {
+            for (int col = 0; col < 8; col++) {
+                board[row][col] = null;
+            }
+        }
+        board[2][4] = Checker.builder().row(2).col(4).side(Side.WHITE).type(CheckerType.KING).build();
+        board[3][3] = Checker.builder().row(3).col(3).side(Side.BLACK).type(CheckerType.REGULAR).build();
+
+        Assert.assertNotNull(board[3][3]);
+
+        Move jump = new Move(2, 4, 5, 1);
+        gameService.makeMove(player1, jump, game);
+
+        Assert.assertNull(board[3][3]);
     }
 
+    @After
+    public void tearDown() {
```

Comment:
since you're using spring boot, i suggest to create a special test context for the app and decouple the prod and test implementation as much as possible. also, please think about separation of unit tests and integration tests - unit test dependencies should be mocked, integration test dependencies require special context with modified beans (e.g. H2 database instead of prod impl's PostgreSQL)